### ISSUE TITLE
Upgrade breaks working code

### ISSUE THREAD

**Issue by jasdefer** [jacqueskang/IpcServiceFramework#40]
When using the current version `<PackageReference Include="JKang.IpcServiceFramework.Server" Version="2.0.2" />`  the following code throws an exception:

```csharp
services.AddIpc()
```
```
System.IO.FileLoadException: 'Could not load file or assembly 'JKang.IpcServiceFramework.Core, Version=2.0.2.1, Culture=neutral, PublicKeyToken=null'. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)'
```

The version `2.0.1.1` still works with the same setup. Do you have any idea what my mistake could be?

**Comment by jacqueskang** [comment]
Should be fixed by v2.0.3
I just uploaded new packages it will take a few minutes to publish & index.

**Comment by jasdefer** [comment]
Perfect, thanks a lot!
